src/Components/Web.JS/src/Platform/Circuits/CircuitStartOptions.ts

```diff
@@ -22,19 +22,22 @@ export interface ReconnectionOptions {
   maxRetries: number;
   retryIntervalMilliseconds: number;
   dialogId: string;
+  reloadOnCircuitRejected: boolean;
 }
 
 export interface ReconnectionHandler {
   onConnectionDown(options: ReconnectionOptions, error?: Error): void;
   onConnectionUp(): void;
+  onConnectionRejected(options: ReconnectionOptions): void;
```

Comment:
This seems like a pretty good idea to me. I'm not 100% sold on reloading automatically by default (we need to at least reason about why that wasn't already our default) but the ability to configure a handler for this scenario seems valuable.